### PR TITLE
Add share-gcp-printer and unshare-gcp-printer commands to util.

### PR DESCRIPTION
**Pull request by jay0lee**

### CODE REVIEW

File: gcp-cups-connector-util/gcp-cups-connector-util.go [hunk 1]
```diff
@@ -521,3 +561,41 @@ func showGCPPrinterStatus(context *cli.Context) {
...
+		fmt.Printf("Failed to share GCP printer %s with %s: %s\n", context.String("printer-id"), context.String("email"), err)
+	} else {
+		fmt.Printf("Shared GCP printer %s with %s\n", context.String("printer-id"), context.String("email"))
+	}
+}
+
+// unshareGCPPrinter unshares a GCP printer.
+func unshareGCPPrinter(context *cli.Context) {
+	config := getConfig(context)
+	gcpConn := getGCP(config)
```
Comment (jay0lee) [inline gcp-cups-connector-util/gcp-cups-connector-util.go]: @jacobmarble wasn't sure what to do here as other functions name this gcp which overwrites gcp/gcp.go namespace preventing me from using gcp.User / gcp.Manager constants. Should I rename this for all functions for consistency?

Comment (jacobmarble) [inline gcp-cups-connector-util/gcp-cups-connector-util.go]: This is fine.

File: gcp-cups-connector-util/gcp-cups-connector-util.go [hunk 2]
```diff
@@ -115,6 +116,45 @@ func main() {
...
+					Name:  "email",
+					Usage: "Group or user to share with.",
+				},
+				cli.StringFlag{
+					Name:  "role",
+					Value: "USER",
+					Usage: "Role granted. user or manager.",
+				},
+				cli.BoolTFlag{
+					Name:  "skip_notification",
```
Comment (jacobmarble) [inline gcp-cups-connector-util/gcp-cups-connector-util.go]: Hyphenate.

File: gcp-cups-connector-util/gcp-cups-connector-util.go [hunk 3]
```diff
@@ -521,3 +561,41 @@ func showGCPPrinterStatus(context *cli.Context) {
...
+		role = gcp.User
+	case "MANAGER":
+		role = gcp.Manager
+	default:
+		fmt.Println("role should be user or manager.")
+		return
+	}
+
+	err := gcpConn.Share(context.String("printer-id"), context.String("email"),
+		role, context.Bool("skip_notification"))
```
Comment (jacobmarble) [inline gcp-cups-connector-util/gcp-cups-connector-util.go]: Hyphenate skip-notification, since it's the command line.

**Pull request merged**